app/pages/lab/workflow.cjsx

```diff
@@ -33,6 +33,9 @@ EditWorkflowPage = React.createClass
     viewQuery = workflow: @props.workflow.id, reload: @state.forceReloader
     @makeHref 'project-classify', viewParams, viewQuery
 
+  canUseTask: (owner, project, task)->
+    task in project.experimental_tools or owner.admin or owner.display_name is 'brian-testing'
```

Comment:
Can we just make this `task in project.experimental_tools`? That'd leave one right way to enable experimental tools.